dev-client/src/translations/en.json

```diff
@@ -7,12 +7,11 @@
     "last_updated": "Last Modified: {{date}} by {{name}}",
     "progress": "Progress: {{progress}}",
     "members": "{{members}} Members",
-    "create": "CREATE SITE",
-    "more_info": "MORE INFO",
-    "soil_id_prediction": "SOIL ID PREDICTION",
-    "ecological_site_prediction": "ECOLOGICAL SITE PREDICTION",
-    "annual_precip_avg": "ANNUAL PRECIPITATION AVG",
-    "elevation": "ELEVATION",
+    "more_info": "More info",
+    "soil_id_prediction": "Soil ID prediction",
+    "ecological_site_prediction": "Ecological site prediction",
+    "annual_precip_avg": "Annual precipitation avg",
+    "elevation": "Elevation",
```

Comment:
question: is it considered best practice to change the case of i18n'd text programmatically? 

i figured it was best to make the strings uppercase in here since it might vary by language whether it makes sense for them to be upper cased. also if the string is used in multiple places the programmer must remember to uppercase it each time

but i also see the advantages of doing it programmatically. i've never worked on a multilingual UI before

Reply:
- If you put the text in all-caps in the JSON file, you lose bits of information. By using CSS or `.toUpperCase()`, we preserve the extra data. 
- Keeping the text in mixed case allows us to easy change it systemwide at a later point.